### DISCUSSION TITLE
v0.145 : Broken download link

### DISCUSSION THREAD

**Discussion by Kikimahe** [seerge/g-helper#1877] [Q&A] [answered]
Hi, 

The download link for v0.145 doesn't work.
`https://github.com/seerge/g-helper/releases/download/v0.145/GHelper.zip`

The empty page indicating "Not found" is displayed. 

![Capture d'écran 2024-01-10 065839](https://github.com/seerge/g-helper/assets/142881210/daea7762-887b-40da-b60e-723b8f58185a)

**Comment by seerge** [comment, marked as answer]
@Kikimahe fixed, thanks for spotting
